iterative/resource_runner.go

```diff
@@ -338,8 +338,14 @@ sudo bash -c 'cat << EOF > /etc/systemd/system/cml.service
   WantedBy=multi-user.target
 EOF'
 
+{{if eq .cloud "azure"}}
 sudo systemctl enable cml.service
 sudo reboot
+{{else}}
+sudo systemctl daemon-reload
+sudo systemctl enable cml.service --now
+{{end}}
+
```

Comment:
```suggestion
```

Awesome! Please remove the newline or add an hyphen so we don't end the file with too many newlines: @restyled-commits will complain.